.github/workflows/deploy-on-release-created.yaml

```diff
@@ -0,0 +1,42 @@
+name: deploy-on-release-created
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  deploy-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout git repo
+        uses: actions/checkout@v2
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Set MATSim version
+        run: mvn versions:set --batch-mode -DnewVersion=${{ github.event.release.tag_name }} -DgenerateBackupPoms=false
```

Comment:
`-DnewVersion=${{ github.event.release.tag_name }}` assumes that the release (and tag) are just numbers (e.g. `13.0`). So far we have used `matsim-x.x` as a tag, but would suggest switching to the (more standard) `x.x` format.